internal/fs/inode/type_cache.go

```diff
@@ -50,47 +58,58 @@ type typeCache struct {
 	//
 	// INVARIANT: entries.CheckInvariants() does not panic
 	// INVARIANT: Each value is of type cacheEntry
-	entries lrucache.Cache
```

Comment:
Is this a major change? Switching
"github.com/jacobsa/util/lrucache"
to 
"github.com/googlecloudplatform/gcsfuse/internal/cache/lru"?

If so, it should be documented.

Reply:
> Is this a major change?

In terms of performance, no. In terms of functionality, a minor change. `jacobsa/util/lrucache` was defined in terms of capacity i.e. upper-limit on the number of entries, but `gcsfuse/internal/cache/lru` is limited by size in MBs. This is to make it consistent with the other cache types (file-read-cache and stat-cache) which also takes sizes instead of capacity as limits.

> If so, it should be documented.

Based on the above, do you think that there is a need to document a change here ?
The change from capacity to size is a user-interface-level change, and that'll be documented.

Reply:
Discussed this offline with @bjornleffler and resolved this issue.